ansible_runner/runner_config.py

```diff
@@ -177,8 +177,8 @@ def open_fifo_write(self, path, data):
         reads data from the pipe.
         '''
         os.mkfifo(path, 0o600)
-        thread.start_new_thread(lambda p, d: open(p, 'w').write(d), (path, data))
-
+        threading.Thread(target=lambda p, d: open(p, 'w').write(d),
+                         args=(path, data))
```

Comment:
Also need to call `start()` on the resulting `Thread` instance.

Reply:
good catch